launch-dev.py

```diff
@@ -11,8 +12,8 @@
 
 sequence = MDASequence(
     channels=["Cy5", {"config": "FITC", "exposure": 50}],
-    time_plan={"interval": 2, "loops": 5},
-    z_plan={"range": 4, "step": 0.5},
+    time_plan={"interval": 2, "loops": 2},
```

Comment:
does this pr require this?

Reply:
no but it saved me 6 seconds every time I would iterate. Will remove at the end